src/Observable.js

```diff
@@ -0,0 +1,69 @@
+var Disposable = require("src/Disposable");
+var Subscriber = require("src/Subscriber");
+
+module.exports = function() {
+    
+    function Observable(source) {
+        this.source = source;
+    }
+    
+    Observable.create = function(subscribe) {
+        return new Observable({ subscribe: subscribe });
+    };
+    
+    Observable.template = function(SubscriberType) {
+        function ObservableTemplate(source) {
```

Comment:
Why are we not just using standard JS sub classing? ObservableTemplate function is created multiple times.